src/styles/blocks/button.scss

```diff
@@ -0,0 +1,40 @@
+.button {
+  position: relative;
+  z-index: 1;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  box-sizing: border-box;
+  height: 56px;
+  width: 100%;
+  font-weight: 700;
+  text-decoration: none;
+  color: #fff;
+  background-color: $colorBlack;
+  border-radius: 40px;
+  cursor: pointer;
+
+  transition: all 0.3s;
+
+  &--primary {
```

Comment:
The class modifier should use double underscore instead of double dash. Use `.button__primary` instead of `.button--primary` to follow the BEM naming convention. 🎨